inst/models/shiny_measles.R

```diff
@@ -315,49 +316,83 @@ measles_panel <- function(model_alt) {
   shiny::conditionalPanel(
     simulate_button("measles"),
     condition = sprintf("input.model == '%s'", model_alt),
-    shiny::numericInput(
-      inputId = "measles_population_size",
-      label   = "Population Size",
-      min     = 0,
-      max     = 50000,
-      value   = 500
+    bslib::tooltip(
+      shiny::numericInput(
+        inputId = "measles_population_size",
+        label   = "Population Size",
+        min     = 0,
+        max     = 50000,
+        value   = 500
+      ),
+      placement = "right",
+      "# of students in the school"
     ),
-    shiny::numericInput(
-      inputId = "measles_prevalence",
-      label   = "Initial cases",
-      value   = 1,
-      min     = 1,
-      max     = NA,
-      step    = 1
+    bslib::tooltip(
+      shiny::numericInput(
+        inputId = "measles_prevalence",
+        label   = "Initial cases",
+        value   = 1,
+        min     = 1,
+        max     = NA,
+        step    = 1
+      ),
+      placement = "right",
+      "# of cases at the start of the simulation"
     ),
-    slider_input_rate(
-      "measles", "Proportion Vaccinated", 0.85,
-      maxval = 1, input_label = "prop_vaccinated"
+    bslib::tooltip(
+      slider_input_rate(
+        "measles", 
+        "Proportion Vaccinated", 
+        0.85,
+        maxval = 1,
+        input_label = "prop_vaccinated"
       ),
-    numeric_input_ndays("measles"),
+      placement = "right",
+      "Proportion of students in the school who are vaccinated against measles"
+    ),
+    bslib::tooltip(
+      numeric_input_ndays("measles"),
+      placement = "right",
+      "# of days to run the simulation"
+    ),
     bslib::accordion(
       open = FALSE,
       bslib::accordion_panel(
         title = "Quarantine",
-        shiny::numericInput(
-          inputId = "measles_days_undetected",
-          label   = "Days Undetected",
-          value   = "2",
-          min     = 0,
-          max     = NA,
-          step    = .5
+        bslib::tooltip(
+          slider_input_rate(
+            "measles", 
+            "Quarantine Willingness", 
+            1.0,
+            maxval = 1, 
+            input_label = "quarantine_willingness"
+          ),
+          placement = "right",
+          "How willing infected people are to quarantine (1 = 100% willing, 0 = 0% willing)"
```

Comment:
Quarantined people are not necessarily infected.
```suggestion
          "How willing people are to stay home from school when asked to quarantine (1 = 100% willing, 0 = 0% willing)"
```